httpd.c

```diff
@@ -128,6 +129,25 @@ char *request_header(const char *name) {
 // get all request headers
 header_t *request_headers(void) { return reqhdr; }
 
+// Handle escape characters (%xx)
+static void uri_unescape(char *s)
```

Comment:
`s` means `raw_string`?

Reply:
s for source (change in src, which should be clearer).